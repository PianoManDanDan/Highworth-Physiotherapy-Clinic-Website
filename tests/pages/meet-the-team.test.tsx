import React from 'react';
import { getPage } from 'next-page-tester';

describe('MeetTheTeam Page', () => {
  test('matches the snapshot', async () => {
    const { render } = await getPage({
      route: '/meet-the-team',
    });

    const page = render();

    expect(page).toMatchSnapshot();
  });
});
