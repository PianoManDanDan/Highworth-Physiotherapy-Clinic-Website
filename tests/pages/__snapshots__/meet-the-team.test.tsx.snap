// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetTheTeam Page matches the snapshot 1`] = `
Object {
  "nextRoot": <div
    id="__next"
  >
    <header>
      <div>
        <a
          href="/"
        >
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAiIGhlaWdodD0iMTI4IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Highworth Physiotherapy Clinic"
                decoding="async"
                src="/assets/logos/hpc.png"
                srcset="/_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=32&q=75 1x, /_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=64&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Highworth Physiotherapy Clinic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </a>
        <div>
          <a
            href="/"
          >
            <h1>
              Highworth Physiotherapy Clinic
            </h1>
          </a>
          <address>
            <h2>
              <a
                href="tel:+441793763814"
              >
                <svg
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                  />
                </svg>
                <span>
                  01793 763814
                </span>
              </a>
            </h2>
          </address>
        </div>
      </div>
      <nav>
        <a
          href="/"
        >
          Home
        </a>
        <a
          href="/facilities"
        >
          Facilities
        </a>
        <a
          href="/physiotherapy"
        >
          Physiotherapy
        </a>
        <a
          href="/animals"
        >
          Animals
        </a>
        <a
          href="/meet-the-team"
        >
          Meet the Team
        </a>
        <a
          href="/faq"
        >
          FAQ's
        </a>
        <a
          href="/contact"
        >
          Contact Us
        </a>
      </nav>
    </header>
    <main>
      <h1>
        MEET THE TEAM
      </h1>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Sonya Nightingale - Physiotherapist"
                decoding="async"
                src="/assets/people/sonya.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fsonya.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fsonya.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Sonya Nightingale - Physiotherapist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Sonya Nightingale
            <!-- -->
             
            <!-- -->
            MCSP
          </h2>
          <p>
            Qualified 
            <!-- -->
            1985
          </p>
          <p>
            Speciality: 
            <!-- -->
            Hands, Animals, Backs, Necks, Head Aches, Knee Pain, Joint Pain, Arthritis, Biomechanics, Sports Injury Rehabilitation, Manual Therapy, Mobilisations and Manipulations, Soft Tissue Massage, Electrotherapy
          </p>
          <p>
            Sonya graduated from Coventry University with graduate diploma in Physiotherapy in 1985. She is the principle physiotherapist at Highworth Physiotherapy Clinic, which she launched in 1989. She started her physiotherapy career working in NHS hospitals and running clinics in GP surgeries before starting her private clinic. She lectures to GP's on physiotherapy procedures and is a Physio First member. Her special interest is the spine and she spends a lot of time treating backs and necks.
          </p>
          <p>
            Sonya also treats animals and was the chair person for ACPAT from 2009-2013 and is a founder member of RAMP and its president from 2017-2020. She runs regular monthly clinics at local venues, including Wickstead Farm Equestrian Center and Rectory Farm, as well as treating small  animals in the clinic. She runs hands on training days for category "B" ACPAT members and lectures at Hartpury College and Liverpool University. She also gives lectures to vets and physiotherapists at international conferences. Sonya was a veterinary physiotherapist at the London 2012 Olympics, where she helped to treat the horses of some of the equestrian teams from all over the world.
          </p>
          <p>
            Sonya has also written research articles on the effectiveness of topical cooling agents of the equine limb.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Emma Overson - Physiotherapist"
                decoding="async"
                src="/assets/people/emma.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Femma.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Femma.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Emma Overson - Physiotherapist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Emma Overson
            <!-- -->
             
            <!-- -->
            MCSP
          </h2>
          <p>
            Qualified 
            <!-- -->
            1994
          </p>
          <p>
            Speciality: 
            <!-- -->
            Pilates, Backs, Necks, Head Aches, Knee Pain, Joint Pain, Arthritis, Biomechanics, Sports Injury Rehabilitation, Manual Therapy, Mobilisations and Manipulations, Soft Tissue Massage, Electrotherapy
          </p>
          <p>
            Emma graduated from Coventry University with a BSc (Hons) degree in Physiotherapy. She spent 7 years working in the NHS before moving into private practice. She specialises in musculoskeletal problems with a particular interest in lower back / pelvic problems and core stability.
          </p>
          <p>
            Emma gained a qualification in teaching Pilates in 2003 from the Australian Physiotherapy and Pilates Institute which she teaches to patients on a 1:1 basis and also in group classes in Highworth. Emma also teaches classes for the over-60s in Shrivenham and Faringdon. These classes are ideal for people who do not wish to lie down on the floor as all of the exercises are carried out in sitting and standing.
          </p>
          <p>
            Emma is a member of Physio First the organisation for private Physiotherapists.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Fiona Hemmings - Physiotherapist"
                decoding="async"
                src="/assets/people/fiona.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Ffiona.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Ffiona.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Fiona Hemmings - Physiotherapist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Fiona Hemmings
            <!-- -->
             
            <!-- -->
            MCSP
          </h2>
          <p>
            Qualified 
            <!-- -->
            1995
          </p>
          <p>
            Speciality: 
            <!-- -->
            Acupuncture, Animals, Backs, Necks, Head Aches, Knee Pain, Joint Pain, Arthritis, Biomechanics, Sports Injury Rehabilitation, Manual Therapy, Mobilisations and Manipulations, Soft Tissue Massage, Electrotherapy
          </p>
          <p>
            Fiona graduated with BSc (Hons) from the University of the West of England, Bristol as a Chartered Physiotherapist in 1995. She worked in the NHS for 8 years specialising in treating musculoskeletal conditions, with a keen interest in sports injuries. Fiona has been practicing acupuncture since 2000 and has worked with the military for a number of years treating tri-service personnel. She is a member of Physio First and the Acupuncture Association of Chartered Physiotherapists.
          </p>
          <p>
            Fiona left the NHS in 2003 to commence work as a private practitioner and develop her career in veterinary physiotherapy. She graduated from the Royal Veterinary College, London having obtained a master's degree in Veterinary Physiotherapy in 2005 and is a "Category A" member of ACPAT.
          </p>
          <p>
            Within the clinic Fiona treats a variety of musculoskeletal conditions, with a keen interest in sports related injuries and also treats canine clients. Outside the clinic, Fiona treats a mix of equine and canine clients.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Diane Turner - Physiotherapist"
                decoding="async"
                src="/assets/people/diane.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fdiane.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fdiane.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Diane Turner - Physiotherapist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Diane Turner
            <!-- -->
             
            <!-- -->
            MCSP
          </h2>
          <p>
            Qualified 
            <!-- -->
            1987
          </p>
          <p>
            Speciality: 
            <!-- -->
            Shoulders, Backs, Necks, Head Aches, Knee Pain, Joint Pain, Arthritis, Biomechanics, Sports Injury Rehabilitation, Manual Therapy, Mobilisations and Manipulations, Soft Tissue Massage, Electrotherapy, Injections
          </p>
          <p>
            Diane graduated from The Royal London Hospital in 1987. She has a MSc in advanced practice and a diploma in sports medicine. She is an extended scope practitioner and a clinical specialist in the diagnosis and treatment of shoulder injuries.
          </p>
          <p>
            Diane is a qualified prescriber, and is highly experienced in administering injections: both in the NHS and in private practice at Highworth Physiotherapy Clinic. She is also involved in the teaching of physiotherapists and has published many articles in the physiotherapy press.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Shani Prosser - Physiotherapist"
                decoding="async"
                src="/assets/people/shani.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fshani.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fshani.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Shani Prosser - Physiotherapist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Shani Prosser
            <!-- -->
             
            <!-- -->
            MCSP
          </h2>
          <p>
            Qualified 
            <!-- -->
            1997
          </p>
          <p>
            Speciality: 
            <!-- -->
            Backs, Necks, Hands, Head Aches, Knee Pain, Joint Pain, Arthritis, Biomechanics, Sports Injury Rehabilitation, Manual Therapy, Mobilisations and Manipulations, Soft Tissue Massage, Electrotherapy
          </p>
          <p>
            Shani graduated from Sheffield University with a BSc (Hons) in Physiotherapy and has developed a wealth of experience from working in both the NHS and private practice. She has worked as a Physiotherapist for St. Peters Welsh Rugby Club and for the Amateur Welsh Rugby Squad. She has returned to work for us after moving away from the area for a couple of years.
          </p>
          <p>
            In her free time she enjoys swimming, cycling, squash and walking.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Joe Baker - Physiotherapist"
                decoding="async"
                src="/assets/people/joe.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fjoe.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fjoe.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Joe Baker - Physiotherapist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Joe Baker
            <!-- -->
             
            <!-- -->
            MCSP
          </h2>
          <p>
            Qualified 
            <!-- -->
            2012
          </p>
          <p>
            Speciality: 
            <!-- -->
            Domiciliary visits, Care of the elderly
          </p>
          <p>
            Joe graduated from the University of the West of England, Bristol in 2012. He specialises in treating elderly patients and does domiciliary visits for patients that are unable to travel to the clinic. Joe visits us on a consultancy basis by request only.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Steve Martin - Practice Manager"
                decoding="async"
                src="/assets/people/steve.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fsteve.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fsteve.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Steve Martin - Practice Manager"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Steve Martin
            <!-- -->
             
          </h2>
          <p>
            Steve is the Practice Manager, receptionist and general dogs body. He manages the every day running of the clinic, from purchasing stock to cleaning up after the physios.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Debbie Davies - Receptionist"
                decoding="async"
                src="/assets/people/debbie.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fdebbie.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fdebbie.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Debbie Davies - Receptionist"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Debbie Davies
            <!-- -->
             
          </h2>
          <p>
            Debbie is one of our receptionists and welcomes our patients into the clinic as well as helping with general administration.
          </p>
        </div>
      </div>
      <div>
        <div>
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYwIiBoZWlnaHQ9IjE2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Daniel Martin - Web Designer"
                decoding="async"
                src="/assets/people/daniel.jpg"
                srcset="/_next/image?url=%2Fassets%2Fpeople%2Fdaniel.jpg&w=256&q=75 1x, /_next/image?url=%2Fassets%2Fpeople%2Fdaniel.jpg&w=384&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Daniel Martin - Web Designer"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </div>
        <div>
          <h2>
            Daniel Martin
            <!-- -->
             
          </h2>
          <p>
            Daniel designed and built the website, and keeps it up to date as well as managing our social media pages.
          </p>
        </div>
      </div>
    </main>
    <footer>
      <div>
        <div>
          <p>
            Find us on Social Media
          </p>
          <div>
            <a
              href="https://www.facebook.com/HighworthPhysio/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="32"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z"
                />
              </svg>
            </a>
          </div>
        </div>
        <div>
          <p>
            All of our physios are registered with at least one of the following regulatory bodies:
          </p>
          <ul>
            <li>
              <a
                href="https://www.csp.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Chartered Society of Physiotherapists"
                >
                  CSP
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.hcpc-uk.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Health and Care Professions Council"
                >
                  HCPC
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.physiofirst.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                Physio First
              </a>
            </li>
            <li>
              <a
                href="https://www.aacp.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Acupuncture Association of Chartered Physiotherapists"
                >
                  AACP
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.acpat.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Association of Chartered Physiotherapists in Animal Therapy"
                >
                  ACPAT
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.rampregister.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Register of Animal Musculoskeletal Practitioners"
                >
                  RAMP
                </abbr>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </footer>
  </div>,
}
`;
